### PR TITLE
include-what-you-use: new package

### PR DESCRIPTION
**Pull request by kmilos**

### CODE REVIEW

**Comment by 1480c1** [comment]
This would be nice

File: mingw-w64-include-what-you-use/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Maintainer: Alexey Pavlov <alexpux@gmail.com>
```
Comment (MehdiChinoune) [inline mingw-w64-include-what-you-use/PKGBUILD:1]: Is this your name?

File: mingw-w64-include-what-you-use/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  if check_option "debug" "n"; then
+    extra_config+=("-DCMAKE_BUILD_TYPE=Release")
+  else
+    extra_config+=("-DCMAKE_BUILD_TYPE=Debug")
+  fi
+
+  if [[ ${MINGW_PACKAGE_PREFIX} != *-clang-* ]]; then
+    CC=clang
+    CXX=clang++
+  fi
```
Comment (MehdiChinoune) [inline mingw-w64-include-what-you-use/PKGBUILD:35]: those are exported by makepkg-mingw by default!

File: mingw-w64-include-what-you-use/PKGBUILD (hunk 1 above)
[the remark this answers is not in the document]
Comment (kmilos) [reply mingw-w64-include-what-you-use/PKGBUILD]: Thanks, copy/paste gotcha...

**Pull request merged**